--- origsrc/libshout-2.2.2/src/Makefile.am	2006-01-30 19:42:42.000000000 -0600
+++ src/libshout-2.2.2/src/Makefile.am	2007-04-15 01:35:22.812500000 -0500
@@ -18,7 +18,7 @@
 SUBDIRS = avl net timing httpp $(MAYBE_THREAD)
 
 lib_LTLIBRARIES = libshout.la
-libshout_la_LDFLAGS = -version-info 5:0:2
+libshout_la_LDFLAGS = -version-info 5:0:2 -no-undefined
 
 EXTRA_DIST = theora.c speex.c
 noinst_HEADERS = shout_ogg.h shout_private.h util.h
